docs/source/gee.rst

```diff
@@ -103,6 +106,8 @@ Families
 The distribution families are the same as for GLM, currently implemented are
 
 .. module:: statsmodels.genmod.families.family
+   :synopsis: Generalized Linear Model (GLM) link families
```

Comment:
drop word `link`   
families are the distributions, 
links are for the nonlinear specification of the mean function